pkgs/development/libraries/pico-sdk/default.nix

```diff
@@ -1,4 +1,5 @@
-{ lib, stdenv, fetchFromGitHub, cmake }:
+{ lib, stdenv, fetchFromGitHub, cmake
+, minimal ? false, }:
```

Comment:
Instead of this singular minimal flag, individual feature flags for USB and wireless functionality should probably be provided if possible. Or maybe a way to specify which submodules you want.

Reply:
Choosing each submodule might create a combinatorial explosion of possible hashes.